apps/portal/src/app/connect/pay/onramp-providers/page.mdx

```diff
@@ -12,14 +12,31 @@ export const metadata = createMetadata({
 
 # Onramp Providers
 
-thirdweb Universal Bridge integrates with Stripe, Kado and Transak to power our onramp. By default, we choose a recommended provider based on the location of the user, KYC status, and currency.
+**Onramp Providers** are services that allow users to buy crypto with fiat. You can set preferred providers for your application in the dashboard.
```

Comment:
I don't think you can set it in the dashboard, but you can in the code

Reply:
meant code- woops